turbinia/evidence.py

```diff
@@ -614,7 +612,10 @@ def _preprocess(self, _, required_states):
         path_specs, self.partition_location)
     if path_spec:
       self.path_spec = path_spec
-      self.type_indicator = path_spec.type_indicator
+    else:
+      log.error(
+          'Could not find path_spec for location {0:s}'.format(
+              self.partition_location))
```

Comment:
Do we need to bail out here in this condition?

Reply:
I was considering that. It shouldn't really come up, because if the partition evidence exists there should also be a path_spec. I guess it would be better to raise an exception instead. I'll update it.